### ISSUE TITLE
BuildWatcher is broken in 2.28.6

### ISSUE THREAD

**Issue by tmm1** [fastlane/fastlane#9024]
https://github.com/fastlane/fastlane/pull/8962#pullrequestreview-35456897

```
NoMethodError: [!] undefined method `processed?' for #<Spaceship::TestFlight::Build:0x007fb066700ee0>
gems/fastlane-2.28.6/fastlane_core/lib/fastlane_core/build_watcher.rb:15:in `block in wait_for_build_processing_to_be_complete'
```

**Comment by mpirri** [comment]
PR to fix this: https://github.com/fastlane/fastlane/pull/9025

**Comment by mpirri** [comment]
@tmm1 Thanks so much for bringing this to our attention! The fix was released in _fastlane_ 2.28.7.  When you have a spare moment, can you verify that this is working better for you? 🙌

**Comment by jaimeagudo** [comment]
I experienced the same issue as @tmm1 on 2.28.6, the fix on 2.28.7 is working for me. Many thanks for the super fast fix @mpirri 👍

**Comment by mpirri** [comment]
I'm optimistic that this issue is resolved, so I'm going to close this ticket now. @tmm1 if you are still having problems, let us know and we can dig deeper.

Thanks!
